codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -231,7 +231,7 @@ void handlerAdded0(@SuppressWarnings("unsed") ChannelHandlerContext ctx) throws
      */
     @Override
     public final void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
-        if (evt instanceof Http2ConnectionPrefaceWrittenEvent) {
+        if (evt instanceof Http2ConnectionPrefaceAndSettingsFrameWrittenEvent) {
```

Comment:
`if evt == Http2ConnectionPrefaceAndSettingsFrameWrittenEvent.INSTANCE`

Reply:
good point since this is the only instance of this class. changed.